src/validation/error.messages.ts

```diff
@@ -180,6 +181,21 @@ export enum ErrorMessages {
   CEASED_DATE_BEFORE_FILING_DATE = "Ceased date must be before or on the date of the update statement",
   RESIGNED_ON_BEFORE_FILING_DATE = "Resigned on date must be before or on the date of the update statement",
   DATE_AFTER_MADE_UP_TO_DATE = "The date you provide must be on or before %s",
+  DAY_OF_CEASED_TRUST_INDIVIDUAL = "The date they ceased being involved in the trust must include a day",
+  DAY_LENGTH_OF_CEASED_TRUST_INDIVIDUAL = "The day of the date they ceased being involved in the trust must include 1 or 2 numbers",
+  MONTH_OF_CEASED_TRUST_INDIVIDUAL = "The date they ceased being involved in the trust must include a month",
+  MONTH_LENGTH_OF_CEASED_TRUST_INDIVIDUAL = "The month of the date they ceased being involved in the trust must include 1 or 2 numbers",
+  YEAR_OF_CEASED_TRUST_INDIVIDUAL = "The date they ceased being involved in the trust must include a year",
+  YEAR_LENGTH_OF_CEASED_TRUST_INDIVIDUAL = "The year of the date they ceased being involved in the trust must include 4 numbers",
+  ENTER_DATE_OF_TRUST_CEASED_INDIVIDUAL = "Enter the date they ceased being involved in the trust",
```

Comment:
See this comment - https://github.com/companieshouse/overseas-entities-web/pull/1426/files#r1648895691.

I think `_CEASED_INDIVIDUAL` can be removed from the end of these error message constants and then error messages can be shared across the implementations.

Reply:
will do, if any mention 'individual' then i'll leave them but can remove for others.

Reply:
i'm changing them to end _CEASED_TRUSTEE
eg
DAY_LENGTH_OF_CEASED_TRUSTEE

Reply:
updated